roles/kubespray-defaults/vars/main.yml

```diff
@@ -4,6 +4,6 @@
 kube_proxy_deployed: "{{ 'addon/kube-proxy' not in kubeadm_init_phases_skip }}"
 
 # The lowest version allowed to upgrade from (same as calico_version in the previous branch)
-calico_min_version_required: "v3.19.4"
+calico_min_version_required: "3.19.4"
```

Comment:
```suggestion
calico_min_version_required: "v3.27.5"
```

Reply:
Since that we have only hashes starting from v3.27.x

Reply:
I'm not completely sure what the original intent of the check is. Was there some historical problem with upgrading from older calico version ?

Reply:
The oldest version in the previous release 2.27 is `v3.27.0`